resource-directory/config.yaml

```diff
@@ -25,6 +25,8 @@ apis:
           useSystemCAPool: false
 clients:
   eventBus:
+    # number of routines to process events in projection
+    goPoolSize: 16
```

Comment:
Why you added it only to RD and not RA ?
Please update also documentation Trishia already did for RD. It's in v2.

Reply:
updated doc - new evenstore doesn't use goPool.